package.json

```diff
@@ -56,6 +56,14 @@
     "directories": {
       "output": "release"
     },
+    "fileAssociations": [
+      {
+        "ext": "appium",
```

Comment:
I wonder if '.appium' is the best extension name. It's really more like an appium session. maybe .appiumsession? What if we, for example, end up saving other things down the road?